### ISSUE TITLE
Vulnerability reported due to optionator v0.9.1 transitive dependency

### ISSUE THREAD

**Issue by padamstx** [stoplightio/spectral#2499]
**Chore summary**
The @stoplight/spectral-cli package has an indirect dependency on optionator v0.9.1, which has a vulnerability ([CVE-2023-26115](https://github.com/advisories/GHSA-j8xg-fqg3-53r7)) due to its "word-wrap" dependency. Optionator v0.9.3 was recently released which fixes this by using a different "word-wrap" package.  

The purpose of this issue is to request that spectral-cli be updated to avoid this CVE.

**Tasks**
- [ ] Upgrade dependencies so that optionator v0.9.3 is used in order to avoid CVE-2023-26115

**Additional context**
n/a

**Comment by Jokinen** [comment]
In our research, we noticed this kind of a dependency path for `word-wrap`.

```
@stoplight/spectral-cli@6.8.0
└─┬ proxy-agent@5.0.0
  └─┬ pac-proxy-agent@5.0.0
    └─┬ pac-resolver@5.0.1
      └─┬ degenerator@3.0.4
        └─┬ escodegen@1.14.3
          └─┬ optionator@0.8.3
            └── word-wrap@1.2.3
```

**Comment by P0lip** [comment]
Should be addressed by #2513 which drops proxy-agent

**Comment by padamstx** [comment]
Thanks for addressing this, @P0lip !
